backend/src/main/java/com/google/backend/servlets/AlertConfigurationServlet.java

```diff
@@ -47,13 +47,23 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    List<String> results = new ArrayList<String>();
-
     Query query = new Query("Configuration");
-    PreparedQuery entities = datastore.prepare(query);
+    PreparedQuery results = datastore.prepare(query);
+
+    List<Configuration> configurations = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      Configuration configuration = new Configuration(
+        (String) entity.getProperty("user"),
```

Comment:
nit: make all these constants since you will need them when you query for properties of the `Configuration`. you can put these constants with the `Configuration.java` file that you put your `Configuration` class in since these constants are all related in the `Configuration` class!
ex. 
```
public final class Configuration { 
  public static final String USER_PROPERTY = "user";
  public static final String DIMENSION_PROPERTY = "dimension";
  ...
}
```
Then when you want to get property of an entity you can do:
```
(String) entity.getProperty(Configuration.USER_PROPERTY);
```
This way when you need to query for configuration, you can make sure you're using the correct property name :)